### PR TITLE
Add Shoot Player and fix Slap command

### PR DESCRIPTION
**Pull request by IllusiveTea**

### CODE REVIEW

File: admin_client.lua [hunk 1]
```diff
@@ -42,7 +43,7 @@ Citizen.CreateThread( function()
 			FreezeEntityPosition(PlayerPedId(), frozen)
 			if IsPedInAnyVehicle(PlayerPedId(), true) then
 				FreezeEntityPosition(GetVehiclePedIsIn(PlayerPedId(), false), frozen)
-			end 
+			end
```
Comment (Bluethefurry) [inline admin_client.lua]: nice unnecesarry change.

File: admin_client.lua [hunk 2]
```diff
@@ -74,12 +75,18 @@ AddEventHandler('EasyAdmin:SlapPlayer', function(slapAmount)
...
+	ShootSingleBulletBetweenCoords(offset1, offset2, shotDamage, nil, GetHashKey("WEAPON_PISTOL"), 0, false, true, 0.25)
+end)
+
 AddEventHandler('EasyAdmin:FreezePlayer', function(toggle)
 	frozen = toggle
 	FreezeEntityPosition(PlayerPedId(), frozen)
 	if IsPedInAnyVehicle(PlayerPedId(), false) then
 		FreezeEntityPosition(GetVehiclePedIsIn(PlayerPedId(), false), frozen)
-	end 
+	end
```
Comment (Bluethefurry) [inline admin_client.lua]: another one

File: admin_server.lua [hunk 3]
```diff
@@ -10,6 +10,7 @@ permissions = {
 	teleport = false,
 	manageserver = false,
 	slap = false,
+	shoot = false,
```
Comment (Bluethefurry) [inline admin_server.lua]: not sure if we actually need a seperate permission for this, considering its the same as slapping someone, only with a bullet

File: gui_c.lua [hunk 4]
```diff
@@ -21,8 +21,10 @@ _menuPool = NativeUI.CreatePool()
 
 -- generate "slap" table once
 local SlapAmount = {}
+local BulletDamage = {}
```
Comment (Bluethefurry) [inline gui_c.lua]: this could also just use the slap amount table, couldn't it?

Comment (IllusiveTea) [reply gui_c.lua]: Yeah, didnt think about that.

**Comment by Bluethefurry** [comment]
locked due to inactivity.

**Pull request closed without merging**